var/spack/repos/builtin/packages/vecgeom/package.py

```diff
@@ -5,49 +5,92 @@
 
 
 from spack import *
-import platform
 
 
-class Vecgeom(CMakePackage):
+class Vecgeom(CMakePackage, CudaPackage):
     """The vectorized geometry library for particle-detector simulation
     (toolkits)."""
 
     homepage = "https://gitlab.cern.ch/VecGeom/VecGeom"
-    url = "https://gitlab.cern.ch/api/v4/projects/VecGeom%2FVecGeom/repository/archive.tar.gz?sha=v0.3.rc"
+    url = "https://gitlab.cern.ch/VecGeom/VecGeom/-/archive/v1.1.5/VecGeom-v1.1.5.tar.gz"
+    git = "https://gitlab.cern.ch/VecGeom/VecGeom.git"
 
-    version('01.01.03', git='https://gitlab.cern.ch/VecGeom/VecGeom.git', tag='v01.01.03', preferred=True)
-    version('01.00.00', git='https://gitlab.cern.ch/VecGeom/VecGeom.git', tag='v01.00.00')
-    version('00.05.00', git='https://gitlab.cern.ch/VecGeom/VecGeom.git', tag='v00.05.00')
+    maintainers = ['sethrj']
+
+    version('master', git=git)
+    version('1.1.6b1', commit='f844ee504f97d8582bffb82d4bcdb85e54c13b20')
+    version('1.1.5', sha256='da674f3bbc75c30f56c1a2d251fa8930c899f27fa64b03a36569924030d87b95')
+    version('1.1.3', tag='v01.01.03')
+    version('1.0.1', sha256='1eae7ac9014c608e8d8db5568058b8c0fea1a1dc7a8f54157a3a1c997b6fd9eb')
+    version('0.5.2', tag='v00.05.02',
+            commit='a7e0828c915ff936a79e672d1dd84b087a323b51')
     version('0.3.rc', sha256='a87a9ea4ab126b59ff9c79182bc0911ead3d76dd197194742e2a35ccd341299d')
 
-    variant('cxxstd',
-            default='17',
-            values=('11', '14', '17'),
-            multi=False,
-            description='Use the specified C++ standard when building.')
-    variant('vector',
-            default='native',
-            values=('sse3', 'sse4.2', 'native'),
-            multi=False,
-            description='Specify the instruction set for vectorization.')
+    _cxxstd_values = ('11', '14', '17')
+    variant('cxxstd', default='14', values=_cxxstd_values, multi=False,
+            description='Use the specified C++ standard when building')
+    variant('gdml', default=False,
+            description='Support Geant XML geometry descriptions')
```

Comment:
The odd thing is that the `gdml` here (corresponding to the `GDML` cmake option) isn't necessary to load GDML files: VecGeom can load them through ROOT with this option disabled.  @pcanal @mrguilima Can you explain the `GDML` cmake option in VecGeom? Is it for loading GDML files *independently* of both ROOT and Geant4?

Reply:
AFAIK, this option builds the `libvgdml` library, that's implemented [here](https://gitlab.cern.ch/VecGeom/VecGeom/tree/master/persistency/gdml). So I'd guess independent of ROOT/Geant4 implementations?

Reply:
Yes, this new feature/library is adding parsing/loading of GDML file to VecGeom without having to rely on ROOT.

Reply:
Thanks @pcanal! @sethrj I think based on that the "Support GDML XML geometry descriptions" fix is appropriate, but otherwise no other changes as the `xerces-c` dependency is in place.